private/pkg/spdx/cmd/spdx-proto-data/main.go

```diff
@@ -0,0 +1,114 @@
+// Copyright 2020-2022 Buf Technologies, Inc.
```

Comment:
I was thinking about should I check in this tool, as it is pretty one-off and didn't expect to regenerate with newer version of SPDX list, as the enum will break because of the new licenses (we might make this tool to handle that case, but it doesn't seems like that is something we care for now? the last update of the version here is more than a year ago), but I think it's still good to have some review on this, especially on the replacer, and we can exclude this if we have thoughts about that

Reply:
I think like you point out, managing this list over time becomes difficult as you have to keep existing identifiers on their proto tags and can't just re-run this script. Any update would have to undo the stringing to get back the identifier (or take them from the comments and any reserved fields), match that against the new list, change the missing ones into "reserved", add new ones to the end.

All of this gives us strict enums over the wire, but it seems like both the frontend and the backend will have to map them back and forth to the spdx identifier strings (string in yaml file, some enum on the frontend to get the full name, etc). 

Given that the strings are well identified according to some external specification, maybe it will be more straightforward to not type this as an enum and use a the well defined identifiers as string?

Reply:
I think it's sane to use the identifier string over the wire, consider that both end need to map the enums back to string to make it usable, and it would be much easier for us to manage the list.
we might need to do some validation in both end but we can defer this implementation detail later.
updated.